client/src/Components/Common/Header/Burgermenu.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import clsx from 'clsx';
+import './icons.css';
+import { Link } from 'react-router-dom';
+import { withStyles } from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import MenuIcon from '@material-ui/icons/Menu';
+import IconButton from '@material-ui/core/IconButton';
+import Icon from '@material-ui/core/Icon';
+import Styles from './style';
+class BurgerMenu extends React.Component {
+  state = {
+    right: false,
+    userLogin: [

```

Comment:
I like the approach but I don't think this data needs to be in the component state because I don't think you will change it. I would move userLogin, userNotLogin, and userAdmin a separate file like Burgermenu.config.js and export them with clear names like `userMenuItems`, `anonymousMenuItems`, `adminMenuItems`, something like that

Reply:
done